workshops/admin.py

```diff
@@ -19,3 +20,4 @@
 admin.site.register(Lesson)
 admin.site.register(KnowledgeDomain)
 admin.site.register(Badge)
+admin.site.register(Certificate)
```

Comment:
That was just for testing purposes, I still need to clean up and add tests, sorry about that!